Automattic/WP-Job-Manager
# Fix for the hidden and displayed datepicker fields not holding the cu…

Fix for the hidden and displayed datepicker fields not holding the current values for _job_expires in datepicker.js.

Fixes #2739

### Changes Proposed in this Pull Request
* Properly updates the inputs selected with these selectors: "[name='_job_expires']" and "input[name='_job_expires-datepicker']" with the correct values on the edit page.


### Testing Instructions
* Go to a job edit page and confirm that the value in the field is not blank and is set to the correct value.
* Look in the dev console to make sure that the hidden field also has the correct value (rather than its value being empty).



### Release Notes
* Fix for the hidden and displayed datepicker fields not holding the current values for _job_expires in datepicker.js.
